get_data_from_cvs.sh

```diff
@@ -26,26 +24,26 @@ curl \
   -H "$(cat ${currentDirectory}/cookie-data.txt)" \
   -H 'TE: Trailers' > ${currentDirectory}/cvs-output.json
 
-for t in ${towns}
+cat ${currentDirectory}/towns.txt | while read t
 do
 
   areAnyAvailable=$(
-    jq --arg t ${t} \
+    jq --arg t "${t}" \
       -r \
       '.responsePayloadData.data.NY[] | select(.city == $t) | .status == "Available"' \
       -- ${currentDirectory}/cvs-output.json
   )
 
   if [[ "${areAnyAvailable}" == 'true' ]]
   then
-    echo "Yes vaccines available at ${t} CVS! $(date)"
+    echo "Yes vaccines available at "${t}" CVS! $(date)"
     curl -X POST https://textbelt.com/text \
       --data-urlencode phone=${phoneNumber} \
-      --data-urlencode message="There are vaccines available at ${t} CVS! $(date) ${linkToWebsite}" \
+      --data-urlencode message="There are vaccines available at "${t}" CVS! $(date)" \
```

Comment:
Can you please leave the link to the website here?

Reply:
You bet. Sorry about that!